lib/ex_unit/lib/ex_unit/capture_log.ex

```diff
@@ -99,6 +99,8 @@ defmodule ExUnit.CaptureLog do
     case :proc_lib.start(__MODULE__, :init_proxy, [pid, opts, self()]) do
       :ok ->
         :ok
+      :noproc ->
+        raise "Can't `capture_log` if the `logger` app is not started"
```

Comment:
Error messages start with lowercase and we don't use backticks in Elixir messages. I would try this:

    raise "cannot capture_log/2 because the :logger application was not started"